app/models/manageiq/providers/ansible_tower/configuration_manager/playbook.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook < ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript
+  belongs_to  :configuration_project, class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationProject"
+  has_many    :configuration_script,  class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript"
```

Comment:
So this relations only makes sense to an Ansible `JobTemplate` (our `ConfigurationScript`) and Ansible `Playbook` (our `ConfigurationScriptBase/Template`).   Other configuration managers may not have this relations (e.g. Foreman).  Do we still want it to be in base class?

Also, our base class name `ConfigurationScript` == `JobTemplate` is weird to me.  A `Script` is more reminiscent of a `Playbook` than a `Template`.